Makefile

```diff
@@ -1,55 +1,34 @@
 DJANGO_PORT = 9001
+DJANGO_SETTINGS_MODULE = hypha.settings.dev
 JS_VENDOR_DIR = ./hypha/static_src/javascript/vendor
 JS_ESM_DIR = ./hypha/static_src/javascript/esm
 
+
 .PHONY: help
-help:
-	@echo "Usage:"
-	@echo "  make help               prints this help."
-	@echo "  make build              build js and css resources for development"
-	@echo "  make cov-html           generate html coverage report"
-	@echo "  make lint               run css, js and python linting."
-	@echo "  make fmt                run code formatters on all code."
-	@echo "  make lint-fix           try fixing plausible python linting issues."
-	@echo "  make py-test            run all python tests and display coverage"
-	@echo "  make test               run linting and test and generate html coverage report"
-	@echo "  make serve-docs         run documentation development server"
-	@echo "  make serve-django       run Django development server on port 9001."
-	@echo "  make serve              run Django and docs preview server, also watch and compile frontend changes"
-	@echo "  make watch              watch js and css resources for development"
-	@echo "  make download-esm-modules  download esm modules from npm and copy to static_src"
-	@echo "  make clean-test-files  remove test files during test"
+help: ## Show this help
+	@echo "\nSpecify a command. The choices are:\n"
+	@grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "  \033[0;36m%-12s\033[m %s\n", $$1, $$2}'
+	@echo ""
+
 
 .PHONY: serve
-serve:
-	$(MAKE) -j3 watch serve-django serve-docs
+serve: .cache/tandem .cache/py-packages .cache/dev-build  ## Run Django and docs preview server, also watch and compile frontend changes
+	@.cache/tandem \
+		'python manage.py runserver 0.0.0.0:$(DJANGO_PORT) --settings=$(DJANGO_SETTINGS_MODULE)' \
```

Comment:
Switch to `runserver_plus`? We already have "Werkzeug" installed. Nice error handling!

Should we also switch to `127.0.0.1` instead of `0.0.0.0`? Do devs ever access the server from other devices on the LAN?